src/ms-review.tex

```diff
@@ -230,14 +230,22 @@ \section{Introduction}
 
 
 %% \gammaray astronomy
-%% CN: I think we should use the expression Water Cherenkov Detectors and use the
-%% abbreviation (WCD) all along the text, later we use "particle samplers"
-The \gammaray range of the electromagnetic spectrum provides us insights into the
-most energetic processes in the universe such as those accelerating particles in the surroundings of
-black holes, and remnants of supernova explosions. As in other
-branches of astronomy, \gammarays can be observed by
-satellite as well as ground-based instruments.
-Ground-based instruments use the Earth's atmosphere as a particle detector.
+In modern astronomy astrophysical objects are studied across
+all wavelenghts. The \gammaray range of the electromagnetic spectrum 
+provides us insights into the most energetic processes in the universe such as
+those accelerating particles in the surroundings of black holes or remnants of
+supernova explosions. 
+
+As in other branches of astronomy, \gammarays can be observed by satellite as
```

Comment:
Maybe it could useful to recall that gamma-ray astronomy is based on individual event detection (and direction measurement)